internal/config/controller.go

```diff
@@ -796,6 +796,8 @@ func (c *controller) detectRemovedStableComponents(comps map[string]*s2hv1.Compo
 func (c *controller) getCronJobSuffix(schedule string) string {
 	suffix := strings.Replace(schedule, " ", "x", -1)
 	suffix = strings.Replace(suffix, "*", "", -1)
+	suffix = strings.Replace(suffix, "/", "", -1)
+	suffix = strings.Replace(suffix, ",", "", -1)
```

Comment:
should we replace `/`, `,` with different char, to prevent duplicate name?

Reply:
also could we add test?

Reply:
updated, and not sure do we need to test this function since this is only used for generating job suffix

Reply:
add unit test